test/debezium/mysql/40-check-smoke.td

```diff
@@ -7,6 +7,11 @@
 # the Business Source License, use of this software will be governed
 # by the Apache License, Version 2.0.
 
+# This test is broken.
+# See: https://github.com/MaterializeInc/materialize/issues/13629
```

Comment:
Can you add a link to this test in the GH issue? Otherwise I'm worried it will get forgotten about.

Reply:
Good catch! Done.